lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -536,3 +536,12 @@ void swift::ide::printCodeCompletionResultFilterName(
     }
   }
 }
+
+StringRef swift::ide::getCodeCompletionResultFilterName(
+    const CodeCompletionString *Str, llvm::BumpPtrAllocator &Allocator) {
+  SmallString<32> buf;
+  llvm::raw_svector_ostream OS(buf);
+  printCodeCompletionResultFilterName(Str, OS);
+  // Use 'copyCString' to get a null terminated StringRef.
+  return {copyCString(Allocator, buf), buf.size()};
```

Comment:
Why does it need to be null terminated here?

Also, the usual nitpick of lower vs upper case for the first letter 😅 (quite a few places like this). Honestly at this point though unless we add a formatter and use it in CI, I think I'll just stop pointing it out.

Reply:
I actually prefer 'lowerCamelCase' for local variable 😓

Reply:
I actually prefer `lowerCamelCase` for local variables 😓

Reply:
Me too. I don't know, we're ostensibly meant to be using the LLVM style guide but... there's no CI for it sooo 🤷